### DISCUSSION TITLE
Format de date affiché

### DISCUSSION THREAD

**Discussion by ChrisThe01** [MTESSDev/FRW#151] [Annonces]
Dans nos formulaires, les champs de date sont affichés comme suit:
![image](https://github.com/MTESSDev/FRW/assets/129303869/b001fe5f-1cea-4b4d-9d21-98de0282018d)
Nous voudrons que l'affichage ce fait comme aaaa--mm-jj au lieu de aaaa-mm-DD

**Comment by LarcM7** [comment]
Bonjour,

Le contrôle de date est celui natif du navigateur.

![image](https://github.com/MTESSDev/FRW/assets/129791924/a946ef4a-156a-48b6-92fa-291a914acb33)

**Comment by anisite** [comment]
J'ai ouvert un incident chez Microsoft pour leur faire corriger. #MA13FJ
